pheweb/load/top_hits.py

```diff
@@ -53,7 +53,7 @@ def run(argv):
 just the top phenotype for each, use `pheweb top-loci`.
 '''.format(out_filepath_json,
            out_filepath_tsv,
-           '{:0.0e}'.format(conf.top_hits_pval_cutoff).replace('e-0', 'e-'),
+           '{:0.0e}'.format(min(conf.top_hits_pval_cutoff, conf.manhattan_peak_pval_threshold)).replace('e-0', 'e-'),
```

Comment:
I may have missed where the manhattan option comes into play here? Here is the function used by `get_all_hits` (called on line 62 here):

https://github.com/statgen/pheweb/blob/1d7d25c7b008a55254195b56f79b3f089e206df3/pheweb/load/top_hits.py#L17

Mainly asking as a check on my own understanding- I'm still finding some of what the pipeline does. AFAIK the `manhattan_peak_pval_threshold` only gets used in manhattan.py.